packages/react-native-bottom-tabs/android/src/main/java/com/rcttabview/RCTTabView.kt

```diff
@@ -384,6 +384,10 @@ class ReactBottomNavigationView(context: Context) : LinearLayout(context) {
     updateTextAppearance()
   }
 
+  fun shutdownImageLoader() {
```

Comment:
nit: Can we call this `onDropViewInstance`? If in the future there is some other deinit code then we will have universal function to do so.

Reply:
Of course! Will fix asap